.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "svn.ignoreMissingSvnWarning": true
```

Comment:
Anything related to vscode should be added to the gitignore

Reply:
removed